### PR TITLE
Add refId to the orderForm query

### PR DESCRIPTION
**Pull request by kaisermann**
#### What problem is this solving?

Dependent of:
- https://github.com/vtex/checkout-graphql/pull/57

Related to:
- https://app.clubhouse.io/vtex/story/33546/add-skurefid-to-pixel-cart-events

#### How should this be manually tested?

[Workspace](url)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

**Review by iaronaraujo — DISMISSED** [review]

**Review by emersonlaurentino — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
```
Comment (kevinch) [inline CHANGELOG.md:9]: Nitpick: missing blank line between titles.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- Add `refId` to the `orderForm` query.
```
Comment (kevinch) [inline CHANGELOG.md]:
```suggestion
- `refId` to the `orderForm` query.
```
Already in the `Added` section.

File: CHANGELOG.md (hunk 1 above)
Comment (kaisermann) [reply CHANGELOG.md:9]: This is something @klzns talked about in a FEG meeting. Sometimes the blank line creates a unneeded conflict in github :(

Comment (kevinch) [reply CHANGELOG.md:9]: Did not know that. Just a nitpick anyway.

**Review by emersonlaurentino — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (marcoskwkm) [reply CHANGELOG.md:9]: I prefer no blank lines, but it seems prettier adds those blank lines automatically if you enable it for `.md` files. Is it possible to change VTEX's prettier rules to treat this case?

**Review by marcoskwkm — APPROVED** [review]

**Pull request merged**